about.html

```diff
@@ -0,0 +1,159 @@
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <link rel="stylesheet" href="css/styles.css">
+  <title>CSS Summit</title>
+</head>
+
+<body>
+  <aside>
+    <ul class="d-flex justify-content-end list-inline container my-1">
+      <li class="mr-2"><a href="" class="aside mr-2"><img src="images/facebook.svg" alt="A link to our facebook page"
+            id="facebook"></a></li>
+      <li class="mr-2"><a href="" class="aside"><img src="images/twitter.svg" alt="A link to our twitter page"></a></li>
+      <li><a href="" class="aside mx-2 text-decoration-none">English</a></li>
+      <li><a href="" class="aside mx-2 text-decoration-none">My Page</a></li>
+      <li><a href="" class="aside mx-2 text-decoration-none">Logout</a></li>
+    </ul>
+  </aside>
+  <nav class="navbar nav-list navbar-expand-sm navbar-light bg-light">
+    <div class="container logo font-weight-bolder">
+      <a class="navbar-brand d-flex w-auto" href="#"><span>The</span> <img src="/images/css.png" width="50" height="50"
+          alt="" class="align-self-center px-1 mb-sm-3"> <span class="align-self-end mb-2">Summit</span>
+        <span class="position-absolute mt-4">2021</span>
+      </a>
+      <button class="navbar-toggler d-lg-none" type="button" data-toggle="collapse" data-target="#collapsibleNavId"
+        aria-controls="collapsibleNavId" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="collapsibleNavId">
+        <ul class="navbar-nav mr-auto mt-2 mt-lg-0">
+          <li class="nav-item">
+            <a class="nav-link active" href="#">About <span class="sr-only sr-only-focusable">About</span></a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" id="nav" href="#">Program <span class="sr-only sr-only-focusable">About</span></a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Join <span class="sr-only sr-only-focusable">About</span></a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Sponsor <span class="sr-only sr-only-focusable">About</span></a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">News<span class="sr-only sr-only-focusable">About</span></a>
+          </li>
+          <li class="nav-item nwk">
+            <a class="nav-link" href="#"><abbr class="text-decoration-none" title="Networking">NWWK</abbr><span
+                class="sr-only sr-only-focusable">
+                <abbr title="Networking">NWK</abbr></span></a>
+          </li>
```

Comment:
- [ ] Hello. On the navbar, the top bar contains a logo to the left and the top bar contains a list of page links aligned to the right. Please, kindly fix that.
- [ ] Please add a home button to your navbar because when I click on the About page there is no way for me to come back to the Home page.

Reply:
I added a home button to the about page and an about button to the home page and left the rest blank to work on them in my free time since it is not a requirement.